Automattic/woocommerce-payments
# Update transaction details link to use Payment Intent ID instead of Charge ID

Fixes #3122

#### Changes proposed in this Pull Request


Update transaction details link to use Payment Intent ID instead of Charge ID


This PR updates the links that lead to the transaction details page in order to use primarily the Payment Intent ID instead of the Charge ID. This change aims to prevent an unnecessary redirect on the transaction details page.

Pages affected:
- Transactions list
- Order details
  - Link on "Order details" box
  - Link on "Order notes" box





#### Testing instructions





- Perform an order on your test shop
- **Transactions List**
  - Open **Payments > Transactions**
  - Click on the transaction that you have just made
  - It should have an `pi_` identifier on the URL and load the transaction details without a redirect
- **Order details**
  - Open **WooCommerce > Orders**
  - Open on the order that you have just made
  - **Order details box (1)**
    - Click on the Payment Intent id link
    - It should have an `pi_` identifier on the URL and load the transaction details without a redirect
  - **Order notes box (2)** 
    - Click on the Payment Intent id link 
    - It should have a `pi_` identifier on the URL and load the transaction details without a redirect

![image](https://user-images.githubusercontent.com/16882226/180241245-f3b8f5a2-0066-4513-a33b-bc17d21f1c2e.png)

-------------------


**Post merge**
